backend/python/app/resources/create_user_dto.py

```diff
@@ -5,3 +5,16 @@ def __init__(self, first_name, last_name, email, role, password):
         self.email = email
         self.role = role
         self.password = password
+
+    def validate():
+        if self.first_name == null || type(self.first_name) != str:
+            return False
+        if self.last_name == null || type(self.last_name) != str:
+            return False
+        if self.email == null || type(self.email) != str:
+            return False
+        if self.role == null || type(self.role) != str:
+            return False
+        if self.password == null || type(self.password) != str:
+            return False
```

Comment:
A suggestion is to raise an Exception (with a message indicating which field validation failed), this way we can report those details to the user in the API response

Reply:
Hm, I think it might be better to just collect the errors in a list and return it to the jsonify that we return anyways since it's an error we can expect to happen rather than an "exceptional" error if that makes sense. I've added the way I think we could do it in my most recent commit.

Reply:
Ok, sounds good 👍